cli/data.go

```diff
@@ -37,7 +37,7 @@ func (c *AppClient) BinaryData(dst string, filter *datapb.Filter, parallelDownlo
 	}
 
 	if err := makeDestinationDirs(dst); err != nil {
-		return errors.Wrapf(err, "error creating destination directories")
+		return errors.Wrapf(err, "could not create destination directories")
```

Comment:
how do wraps render?

Reply:
They render with a `: ` and the wrapping message preceding. Like:
```
=> viam data export --destination /usr/bin/foo --data-type binary
Error: could not create destination directories: mkdir /usr/bin/foo: operation not permitted
```